<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestDescriptionWindow</class>
 <widget class="QMainWindow" name="RequestDescriptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RequestDescriptionWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_applicationID">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_typeOfFault">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_typeOfFault">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип неисправности:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_customer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Заказчик:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_date">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_more">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_status">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_applicationID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер заявки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_more">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>256</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Оборудование:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_customer">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_equipment">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_customer_worker">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Работник:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_worker">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_report">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчёт</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
